packages/supi/src/install/index.ts

```diff
@@ -837,8 +837,16 @@ async function installInContext (
       })
     }
 
-    if (!opts.lockfileOnly) {
-      await Promise.all(projectsToLink.map(linkBinsOfImporter))
+    const binNames = (await Promise.all(projectsToLink.map(linkBinsOfImporter))).flat()
+    if (opts.global) {
+      const installPackages = projects
```

Comment:
installPackages  => installedPackages

Reply:
I take your advice and put checking inside each linkBins and I think the logic is simpler